.pre-commit-config.yaml

```diff
@@ -1,5 +1,14 @@
 default_language_version:
     python: python3
+ci:
+    autofix_commit_msg: |
+        [pre-commit.ci] auto fixes from pre-commit hooks
+    autofix_prs: true
+    autoupdate_branch: 'pre-commit-autoupdate'
```

Comment:
Updates on a branch named `pre-commit-autoupdate` so it does not change our main branch.